scripts/epa/airdata/criteria_gases/criteria_gases.py

```diff
@@ -0,0 +1,196 @@
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Generates cleaned CSV and template MCF files for the EPA AirData Criteria
+Gases.
+
+Usage: python3 criteria_gases.py <end_year>
+'''
+import csv, os, sys, requests, io, zipfile
+
+POLLUTANTS = [
+    '44201',
+    '42401',
+    '42101',
+    '42602',
+]
+
+START_YEAR = 1980
+
+CSV_COLUMNS = [
+    'Date', 'Site_Number', 'Site_Name', 'Site_Location', 'County', 'Mean',
+    'Max', 'AQI', 'Units', 'Method', 'POC', 'Mean_SV', 'Max_SV', 'AQI_SV'
+]
+
+# Template MCF for StatVarObservation
+TEMPLATE_MCF = '''
+Node: E:EPA_CriteriaGases->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: C:EPA_CriteriaGases->Mean_SV
+measurementMethod: C:EPA_CriteriaGases->Method
+observationDate: C:EPA_CriteriaGases->Date
+observationAbout: E:EPA_CriteriaGases->E0
+observationPeriod: dcs:"P1D"
+value: C:EPA_CriteriaGases->Mean
+unit: C:EPA_CriteriaGases->Units
+
+Node: E:EPA_CriteriaGases->E2
+typeOf: dcs:StatVarObservation
+variableMeasured: C:EPA_CriteriaGases->Max_SV
+measurementMethod: C:EPA_CriteriaGases->Method
+observationDate: C:EPA_CriteriaGases->Date
+observationAbout: E:EPA_CriteriaGases->E0
+observationPeriod: dcs:"P1D"
+value: C:EPA_CriteriaGases->Max
+unit: C:EPA_CriteriaGases->Units
+
+Node: E:EPA_CriteriaGases->E3
+typeOf: dcs:StatVarObservation
+variableMeasured: C:EPA_CriteriaGases->AQI_SV
+measurementMethod: C:EPA_CriteriaGases->Method
+observationDate: C:EPA_CriteriaGases->Date
+observationAbout: E:EPA_CriteriaGases->E0
+observationPeriod: dcs:"P1D"
+value: C:EPA_CriteriaGases->AQI
+'''
+
+# Template MCF for Air Quality Site
+TEMPLATE_MCF_AIR_QUALITY_SITE = '''
+Node: E:EPA_CriteriaGases->E0
+typeOf: dcs:AirQualitySite
+dcid: C:EPA_CriteriaGases->Site_Number
+name: C:EPA_CriteriaGases->Site_Name
+location: C:EPA_CriteriaGases->Site_Location
+containedInPlace: C:EPA_CriteriaGases->County
+airQualitySiteMonitor: C:EPA_CriteriaGases->POC
+'''
+
+
+# Convert to CamelCase (splitting on spaces)
+def get_camel_case(s):
+    if s == '' or s == ' - ':
+        return ''
+    parts = s.lower().split()
+    result = ''
+    for i in range(len(parts)):
+        result += parts[i][0].upper()
+        if len(parts[i]) > 0:
+            result += parts[i][1:]
+    return result
+
+
+def get_pollutant_standard(s):
+    return s.replace(' ', '_').replace('-', '')
+
+
+# Hardcoded for consistency in StatisticalVariable names
+def get_suffix(parameter):
+    if parameter == '44201':
+        return 'Ozone'
+    elif parameter == '42401':
+        return 'SO2'
+    elif parameter == '42101':
+        return 'CO'
+    elif parameter == '42602':
+        return 'NO2'
+    return ''
```

Comment:
nit: if we turn POLLUTANTS into a map, we can skip this?

Reply:
ah sorry this is leftover from when I was still splitting by pollutant standards. updated